src/main/java/com/softserve/app/controller/MainPageController.java

```diff
@@ -0,0 +1,36 @@
+package com.softserve.app.controller;
+
+import com.softserve.app.models.User;
+import com.softserve.app.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class MainPageController {
+
+    private final UserService userService;
+
+
+    @RequestMapping(value = "/", method = RequestMethod.GET)
+    @ResponseBody
```

Comment:
if you use RestController annotation, you don't have to use ResponseBody annotation, please remove it. Change RequestMapping to GetMapping